pkg/controllers/managementuser/rbac/namespace_handler.go

```diff
@@ -239,7 +239,8 @@ func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
 
 		rt, err := n.m.rtLister.Get("", prtb.RoleTemplateName)
 		if err != nil {
-			return false, errors.Wrapf(err, "couldn't get role template %v", prtb.RoleTemplateName)
+			logrus.Warnf("ProjectRoleTemplateBinding %q sets a non-existing role template %q. Skipping.", prtb.Name, prtb.RoleTemplateName)
+			continue
```

Comment:
Q: Should we cleanup such orphaned PRTB's instead/in addition to? It's not like the role template will re-appear on it's own.